code/modules/ninja/suit/n_suit_verbs/ninja_adrenaline.dm

```diff
@@ -4,11 +4,22 @@
 
 	if(!ninjacost(0,N_ADRENALINE))
 		var/mob/living/carbon/human/H = affecting
-		H.SetUnconscious(0)
+		H.SetSleeping(0)
 		H.SetStun(0)
 		H.SetKnockdown(0)
-		H.adjustStaminaLoss(-75)
+		H.SetUnconscious(0)
+		H.adjustStaminaLoss(-200)
+		H.updatehealth()
+		H.update_stamina()
```

Comment:
Ditto regarding adjuststaminaloss's existing stamina updates

Reply:
Imma try and see.

Reply:
After a lot of testing, I'm actually ready to defend the existence of those procs in the PR.
Without them, stamina updates the tick after, rather than immediately.